config/quality.gradle

```diff
@@ -34,8 +36,7 @@ apply plugin: 'com.github.spotbugs'
 
 spotbugs.toolVersion = '3.1.3'
 
-task("spotbugs", type: com.github.spotbugs.SpotBugsTask, dependsOn: "assembleDebug") {
-
+tasks.register("spotbugs", SpotBugsTask) {
```

Comment:
In one of the previous commits, you added `dependsOn: "assembleDebug"` and here you removed it, why?

Reply:
It was redundant. Even after removing the `dependsOn: "assembleDebug"`, the task gets executed (probably some internal dependency) before running the `spotbugs` task. Verified by cleaning the build dir and then running the `spotbugs` task